monkey/monkey_island/cc/services/exploitations/manual_exploitation.py

```diff
@@ -0,0 +1,31 @@
+from dataclasses import dataclass
+from typing import List
+
+from monkey_island.cc.database import mongo
+from monkey_island.cc.services.node import NodeService
+
+
+@dataclass
+class ManualExploitation:
+    hostname: str
+    ip_addresses: List[str]
+    start_time: str
+
+
+def get_manual_exploitations() -> List[ManualExploitation]:
+    monkeys = get_manual_monkeys()
+    return [monkey_to_manual_exploitation(monkey) for monkey in monkeys]
+
+
+def get_manual_monkeys():
+    return [
+        monkey for monkey in mongo.db.monkey.find({}) if NodeService.get_monkey_manual_run(monkey)
+    ]
+
```

Comment:
Why is this mongo query different than the one in the `get_manual_monkey()` that was removed from `monkey_island/cc/services/reporting/report.py`?

Reply:
Because the one who got removed only returned hostname